ci-operator/step-registry/openshift/manifests/cgroupsv1/openshift-manifests-cgroupsv1-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
FYI I'm proposing https://github.com/openshift/release/pull/42478 which modifies these manifests to use Node API.